Editor/Diagnostic/DescriptorExtensions.cs

```diff
@@ -33,5 +33,12 @@ public static bool IsPlatformCompatible(this Descriptor descriptor, BuildTarget
                 return true;
             return descriptor.platforms.Contains(buildTarget.ToString());
         }
+
+        public static bool IsPlatformSpecific(this Descriptor descriptor, BuildTarget buildTarget)
```

Comment:
Just to verify:

The method confirms if only the first entry is matching the platform.

This looks like we would also check if there's exactly only one entry, since otherwise we'd miss the 2nd entry that may match our platform (in which case this would have other semantics than being platform-specific I guess and we'd call IsPlatformCompatible() instead).